api/dataset_config/platinum_genomes/README.md

```diff
@@ -0,0 +1,2 @@
+See [here](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/dataset_config/platinum_genomes) for the other dataset config files.
+(`ui.json` is here because it is only used by this repo.)
```

Comment:
I don't think we need to justify why data explorer-specific config is in the data explorer repo